regulations/static/regulations/js/source/views/drawer/drawer-tabs-view.js

```diff
@@ -17,6 +17,7 @@ var DrawerTabsView = Backbone.View.extend({
 
     idMap: {
         'table-of-contents': '#menu-link',
+        'table-of-contents-secondary': '#table-of-contents-secondary-link',
         'timeline': '#timeline-link',
         'search': '#search-link'
```

Comment:
I was also thinking about putting these relationships on the dom so the js doesn't have to know about all possible pairs of link and section ids.